README.md

```diff
@@ -83,6 +83,39 @@ cd example
 vagrant up --provider=hyperv
 ```
 
+## Variables override
+
+Some properties of the virtual machine and the Proxmox VE installation can be overriden.
+Take a look at `proxmox-ve.json`, object `variables`, to get an idea which values can be
+overridden. Do not override `iso_url` and `iso_checksum` as the `boot_command`s might be
+tied to a specific Proxmox VE version. Also take care when you decide to override `country`.
+
+Create the base box:
+
+```bash
+make build-libvirt VAR_FILE=example.vars.json  # or build-virtualbox or build-hyperv
+```
+
+The following content of `example.vars.json`:
+
+* sets the initial disk size to 128 GB
+* sets the initial memory to 4 GB
+* sets the country to Germany (timezone is updated by Proxmox VE installer) and changes
+  the keyboard layout back to "U.S. English" as this is needed for the subsequent
+  `boot_command` statements
+* uses all default shell provisioners (see [`./provisioners`](./provisioners)) and a
+  custom one for german localisation
+
+  ```json
+  {
+      "disk_size": "131072",
+      "memory": "4048",
+      "step_country": "Ger<wait>m<wait>a<wait>n<wait><enter>",
+      "step_keyboard_layout": "<end><up><wait>",
+      "shell_provisioner_scripts": "provisioners/upgrade.sh,provisioners/network.sh,localisation-de.sh,provisioners/reboot.sh,provisioners/provision.sh"
+  }
+  ```
+
 # Packer boot_command
```

Comment:
what do you think about updating this `# Packer boot_command` section to use these `step_` variables?

Reply:
Ok. What do you have in mind?

```markdown
| step                              | boot_command                                          |
|----------------------------------:|-------------------------------------------------------|
| [...]                             |                                                       |
| type country                      | ```{{user `step_country`}}<tab><wait>```              |
| timezone                          | ```{{user `step_timezone`}}<tab><wait>```             |
| keyboard layout                   | ```{{user `step_keyboard_layout`}}<tab><wait>```      |
| [...]                             |                                                       |
| email                             | ```{{user `step_email`}}<tab><wait>```                |
| [...]                             |                                                       |
| hostname                          | ```{{user `step_hostname`}}<tab><wait>```             |
| [...]                             |                                                       |
```

Reply:
yes, just like that. does it make sense?

Reply:
For me, as I know the code, this indirection is ok.

But as a new user I would prefer to read the concrete values instead.

Reply:
Makes sense. Its good as-is. I will test, merge this.